test/runtests.jl

```diff
@@ -26,7 +26,7 @@ end
     b
 end
 @test typeof(A(1,2)) == A
-@test hash(A(1,2)) == hash(2,hash(1,hash(:A)))
+@test_broken hash(A(1,2)) == hash(2,hash(1,hash(:A)))
```

Comment:
```suggestion
@test hash(A(1,2)) == hash((1,2),hash(:A))
```
I think this would fit the new behaviour

Reply:
I'm not sure, because the type itself is hashed `hash(A)`, not the name `hash(:A)`.
I'm also not sure it is worth testing this level of detail.

Reply:
> I'm not sure, because the type itself is hashed hash(A), not the name hash(:A).

Ah you're right, I missed that that was a part of this

> I'm also not sure it is worth testing this level of detail.

I think it's good to test this, as it prevents someone who intends only to change the code used to generate the hash behaviour from accidentally changing the hash behaviour.